### PR TITLE
Location verify method

### PR DESCRIPTION
**Pull request by sknick-iastate**
This pull request addresses the issues identified in #129.  Initial discussion focused on providing a list of enumerations for the location verify method but after further discussion it was determined that verified location information should be made using a GPS equipped device (i.e. not verified through cameras, site inspection, etc).  It was determined that this would be better achieved by updating the [SpatialVerification](https://github.com/usdot-jpo-ode/wzdx/blob/main/spec-content/enumerated-types/SpatialVerification.md) enumerations to clarify that a GPS equipped device is required for verification and depreciating the `location_verify_method`.

As further justification the `location_verify_method` is at the data source level and wouldn't allow for different verification types for individual road events.  If this is a desired then another attribute should be added at the event level.

So in summary the changes are:
- Deprecate `location_verify_method` in the [RoadEventDataSource](https://github.com/usdot-jpo-ode/wzdx/blob/main/spec-content/objects/RoadEventDataSource.md)
- Redefine the [SpatialVerification](https://github.com/usdot-jpo-ode/wzdx/blob/main/spec-content/enumerated-types/SpatialVerification.md) enumerations to clarify that verified work zone locations should use a GPS enabled device

### CODE REVIEW

**Comment by j-d-b** [comment]
@sknick-iastate let me know what you think of my above approach.

**Comment by sknick-iastate** [comment]
@j-d-b sorry for the delayed response.  I like your approach for changing the verification to a boolean.  I'd like to get feedback at our next specification update meeting then can make the change.

**Review by j-d-b — CHANGES_REQUESTED** [review]
See https://github.com/usdot-jpo-ode/wzdx/pull/199#issuecomment-907206410
